miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/RunRestController.java

```diff
@@ -135,19 +136,39 @@ public void setSecurityManager(SecurityManager securityManager) {
 
   @RequestMapping(value = "/dt", method = RequestMethod.GET, produces = "application/json")
   @ResponseBody
-  public DataTablesResponseDto<RunDto> getDTRuns(HttpServletRequest request, HttpServletResponse response, UriComponentsBuilder uriBuilder)
+  public DataTablesResponseDto<RunDto> dataTable(HttpServletRequest request, HttpServletResponse response, UriComponentsBuilder uriBuilder)
       throws IOException {
     return jQueryBackend.get(request, response, uriBuilder);
   }
 
   @RequestMapping(value = "/dt/project/{id}", method = RequestMethod.GET, produces = "application/json")
   @ResponseBody
-  public DataTablesResponseDto<RunDto> getDTRunsByProject(@PathVariable("id") Long id, HttpServletRequest request,
+  public DataTablesResponseDto<RunDto> dataTableByProject(@PathVariable("id") Long id, HttpServletRequest request,
       HttpServletResponse response, UriComponentsBuilder uriBuilder)
       throws IOException {
     return jQueryBackend.get(request, response, uriBuilder, PaginationFilter.project(id));
   }
 
+  @RequestMapping(value = "/dt/platform/{platform}", method = RequestMethod.GET, produces = "application/json")
+  @ResponseBody
+  public DataTablesResponseDto<RunDto> dataTableByPlatform(@PathVariable("platform") String platform, HttpServletRequest request,
+      HttpServletResponse response, UriComponentsBuilder uriBuilder)
+      throws IOException {
+    PlatformType platformType = PlatformType.valueOf(platform);
+    if (platformType == null) {
+      throw new RestException("Invalid platform type.");
```

Comment:
This will default to returning HTTP 500 (Internal Server Error). Include `Status.BAD_REQUEST` as second constructor parameter to return HTTP 400 instead